pubspec.lock

```diff
@@ -7,7 +7,7 @@ packages:
       name: async
```

Comment:
🔴 
commit the pubspec.lock changes only if you have changed the pubspec.yaml file